test/WebJobs.Script.Tests/HttpWorker/DefaultHttpWorkerServiceTests.cs

```diff
@@ -393,6 +428,24 @@ public void ProcessOutputLogs_Succeeds(HttpScriptInvocationResult httpScriptInvo
             }
         }
 
+        [Theory]
+        [InlineData("someFuntionName", CustomHandlerType.Http, true, "/")]
+        [InlineData("someFuntionName", CustomHandlerType.Http, false, "someFuntionName")]
+        [InlineData("someFuntionName", CustomHandlerType.None, true, "someFuntionName")]
+        [InlineData("someFuntionName", CustomHandlerType.None, false, "someFuntionName")]
+        public void TestPathValue(string functionName, CustomHandlerType type, bool enableForwardingHttpRequest, string expectedValue)
+        {
+            HttpRequest testHttpRequest = HttpWorkerTestUtilities.GetTestHttpRequest();
```

Comment:
you can now test different paths for example : ` HttpRequest testHttpRequest = HttpWorkerTestUtilities.GetTestHttpRequest("localhost:8000/api/foo/duh");` , take host string as input